docs/source/agents.rst

```diff
@@ -0,0 +1,41 @@
+Agents API (Python)
+===================
+
+Agent
+-----
+.. autoclass:: agents.agent.Agent
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+
+ContinuousActionBandit
+----------------------
+.. autoclass:: agents.continuous_action_agent.ContinuousActionBandit
```

Comment:
wait... do we still have those classes?

Reply:
(all of the Agent classes...)

Reply:
I've not messed with the documentation. Wasn't planning on it because I don't want to break Tumay's stuff. Gonna wait for him to port first.

Reply:
Rebase the docs folder to reflect master

Reply:
https://github.com/semiotic-ai/autoagora-agents/commit/7e27ea6ad2227fb5e807179f52ecefe4b4bb7f99